pages/windows/invoke-item.md

```diff
@@ -0,0 +1,29 @@
+# Invoke-Item
+
+> Open files in their respective default programs.
+> This command can only be used through PowerShell.
+> More information: <https://docs.microsoft.com/powershell/module/microsoft.powershell.management/invoke-item>.
+
+- Open a file in its default program:
+
+`Invoke-Item -Path "{{path/to/file}}"`
+
+- Open all files inside a directory:
+
+`Invoke-Item -Path "{{path/to/directory}}\*"`
+
+- Open all PNG files inside a directory:
+
+`Invoke-Item -Path "{{path/to/directory}}\*.png"`
+
+- Open all files inside a directory which contain a specific keyword:
+
+`Invoke-Item -Path "{{path/to/directory}}\* -Include "*{{keyword}}*"`
+
+- Open all files inside a directory except those containing specific keyword:
+
+`Invoke-Item -Path "{{path/to/directory}}\* -Exclude "*{{keyword}}*"`
```

Comment:
```suggestion
`Invoke-Item -Path "{{path\to\directory\*}}" -Exclude "{{*keyword*}}"`
```

Reply:
Same here @SethFalco - the quotes around `path/to/directory/*` have to go to make the glob work as far as I'm aware.

Reply:
The point was less about if they should or shouldn't be there, but just if there's an open quote, I'd expect to see a close one.
But I'll keep in mind in future the quotes/globs thing.